studio-docs/source/dev-graphs.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: Development graphs
+sidebar_title: Local development (dev graphs)
+description: Use Studio features in your development environment
+---
+
+Development graphs (or **dev graphs**) are a special type of graph in Apollo Studio. Unlike a _deployed_ graph (which represents your data graph in deployed environments like staging and production), a dev graph represents your data graph in your own development environment. It's only visible to you, and because it's separate from your deployed graph, you can quickly and safely iterate on its schema without affecting other clients.
```

Comment:
I think I would phrase this as a graph "you can quickly spike out changes to and safely iterate on without creating noise for your teammates".